### ISSUE TITLE
can not build bootable iso 

### ISSUE THREAD

**Issue by QuackDaDuck** [CosmosOS/Cosmos#2401]
Have you checked Github Issues for similar errors?
yes

** VS Output Logs **
Build started...
1>------ Build started: Project: NotlekOS, Configuration: Debug Any CPU ------
1>NotlekOS -> C:\Users\kelto\Desktop\NotlekOS\bin\Debug\net5.0\NotlekOS.dll
1>Message: Executing IL2CPU on assembly
1>Message: Kernel Base: Cosmos.System.Kernel
1>Message: Checking target assembly: C:\Users\kelto\Desktop\NotlekOS\bin\Debug\net5.0\NotlekOS.dll
1>Message : warning : Loading plugs from assembly: Cosmos.Core_Asm, Version=10.0.0.0, Culture=neutral, PublicKeyToken=5ae71220097cb983
1>Message : warning : Loading plugs from assembly: Cosmos.Core_Plugs, Version=10.0.0.0, Culture=neutral, PublicKeyToken=5ae71220097cb983
1>Message : warning : Loading plugs from assembly: Cosmos.Debug.Kernel.Plugs.Asm, Version=10.0.0.0, Culture=neutral, PublicKeyToken=5ae71220097cb983
1>Message : warning : Loading plugs from assembly: Cosmos.System2_Plugs, Version=10.0.0.0, Culture=neutral, PublicKeyToken=5ae71220097cb983
1>Detecting fields for type 'IL2CPU.Debug.Symbols.FIELD_INFO'
1>Detecting fields for type 'IL2CPU.Debug.Symbols.FIELD_MAPPING'
1>Detecting fields for type 'IL2CPU.Debug.Symbols.MethodIlOp'
1>Detecting fields for type 'IL2CPU.Debug.Symbols.Document'
1>Detecting fields for type 'IL2CPU.Debug.Symbols.AssemblyFile'
1>Detecting fields for type 'IL2CPU.Debug.Symbols.Method'
1>Detecting fields for type 'IL2CPU.Debug.Symbols.LOCAL_ARGUMENT_INFO'
1>IL2CPU task took 00:01:21.0727198
1>-g dwarf2 -f elf -o bin\Debug\net5.0\NotlekOS.obj -dELF_COMPILATION -O0 bin\Debug\net5.0\NotlekOS.asm
1>Yasm task took 00:00:09.1446629
1>-Ttext 0x2000000 -Tdata 0x1000000 -e Kernel_Start -o bin\Debug\net5.0\NotlekOS.bin bin\Debug\net5.0\NotlekOS.obj
1>LD task took 00:00:00.9757150
1>Done building project "NotlekOS.csproj".
========== Build: 1 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========

**Screenshots**
![image](https://user-images.githubusercontent.com/101152125/186744281-f2a91890-164d-41dd-bc5c-337d9426b12d.png)

![image](https://user-images.githubusercontent.com/101152125/186744449-36863616-6e36-49ec-909d-742fcddc0636.png)

**Context**
Before posting please confirm that the following are in order
[ ] Both Cosmos VS Extensions are installed
[ ] In the NuGet Package Manager "Include prerelease" is selected
[ ] The Cosmos NuGet package store is selected (NOT nuget.org) in 'Manage NuGet Packages'
[ ] The Cosmos NuGet packages are installed

i want to build a iso for rufus so i selected usb bootable drive

**Comment by quajak** [comment]
The error is caused by the debugger, the generated ISO should still be found under the bin directory.

**Comment by AudioTore** [comment]
Yep go to the bin directory in the project folder. The developer clearly is not wanting to fix these problems. :/

**Comment by keltonxgame** [comment]
ya it dose build but I cant flash it to a drive im using rufus

**Comment by ekeleze** [comment]
same problem here

**Comment by ptobuon** [comment]
![image](https://user-images.githubusercontent.com/98218366/191770544-70ca6221-812f-40c7-91c1-bcb3a8e71748.png)
![image](https://user-images.githubusercontent.com/98218366/191770813-316a3756-f187-4a98-b1b6-1291d659911a.png)
You can't, Cosmos is not supported on rufus. 
Use an alternative like balenaetcher

**Comment by ptobuon** [comment]
btw if you want to build the iso you can only compile it, not start it.

**Comment by keltonxgame** [comment]
dose balenaetcher portibale work?

**Comment by Kiirx** [comment]
Use ventoy it would be easier for you as you may do numerous tests, with flashing your usb evertime, you will lose a lot of time

**Comment by keltonxgame** [comment]
i use vmware for testing but i sometimes want to use on real hard ware
i dont want to use ventoy i just want to flash it

**Comment by ptobuon** [comment]
As I said, use Balena Etcher:
https://etcher.io

**Comment by ADev531** [comment]
We can just close now

**Comment by valentinbreiz** [comment]
Cosmos Gen3 has replaced the IL2CPU-based Gen2 codebase, and `gen3` is now the default branch of this repository. This issue was filed against Gen2, which is no longer developed, so it is being closed as part of the Gen3 cutover (#3088).

If you still hit this on Gen3, please open a fresh issue. The compiler, the build pipeline and the debugging story are all different now, so a new report will get a far more useful answer than reviving this one. Start here: https://cosmosos.github.io/

Gen2 remains on the `gen2` branch, and this issue stays readable.
